src/widgets/Window.cpp

```diff
@@ -393,7 +393,7 @@ void Window::addShortcuts()
         getApp()->windows->forceLayoutChannelViews();
     });
 
-    createWindowShortcut(this, "CTRL+K", [this] {
+    createWindowShortcut(this, "CTRL+K", [] {
```

Comment:
This change should probably be in a separate PR

Reply:
Eh, it was just a very little change that meant to fix an annoying warning from my editor, but sure, I'll revert it.